### PR TITLE
feat(input): change fill default on ionic theme

### PR DESCRIPTION
**Pull request by BenOsodrac**
Issue number: internal

---------

## What is the new behavior?

- Now the fill option default value is outline when using the Ionic theme.
- Updated snapshots.

## Does this introduce a breaking change?

- [ ] Yes
- [x] No

## Other information

- [Input Fill Sample](https://ionic-framework-9fbgwzxss-ionic1.vercel.app/src/components/input/test/fill?ionic:theme=ionic)

### CODE REVIEW

**Review by joselrio — APPROVED** [review]

File: core/src/components/input/test/bottom-content/input.e2e.ts-snapshots/input-full-bottom-outline-ionic-md-ltr-light-Mobile-Chrome-linux.png
Comment (brandyscarney) [inline core/src/components/input/test/bottom-content/input.e2e.ts-snapshots/input-full-bottom-outline-ionic-md-ltr-light-Mobile-Chrome-linux.png:1]: Why is "Email" showing up twice here?

File: core/src/components/input/test/fill/input.e2e.ts-snapshots/input-fill-outline-label-stacked-ionic-md-ltr-light-Mobile-Chrome-linux.png
Comment (brandyscarney) [inline core/src/components/input/test/fill/input.e2e.ts-snapshots/input-fill-outline-label-stacked-ionic-md-ltr-light-Mobile-Chrome-linux.png:1]: Why is "Email" showing up twice here?

File: core/src/components/input/test/fill/input.e2e.ts-snapshots/input-fill-outline-label-stacked-size-large-ionic-md-ltr-light-Mobile-Chrome-linux.png
Comment (brandyscarney) [inline core/src/components/input/test/fill/input.e2e.ts-snapshots/input-fill-outline-label-stacked-size-large-ionic-md-ltr-light-Mobile-Chrome-linux.png:1]: Why is "Email" showing up twice here?

File: core/src/components/input/test/highlight/input.e2e.ts-snapshots/input-outline-custom-highlight-ionic-md-ltr-light-Mobile-Chrome-linux.png
Comment (brandyscarney) [inline core/src/components/input/test/highlight/input.e2e.ts-snapshots/input-outline-custom-highlight-ionic-md-ltr-light-Mobile-Chrome-linux.png:1]: Why is "Email" showing up twice here?

File: core/src/components/input/test/highlight/input.e2e.ts-snapshots/input-outline-invalid-ionic-md-ltr-light-Mobile-Chrome-linux.png
Comment (brandyscarney) [inline core/src/components/input/test/highlight/input.e2e.ts-snapshots/input-outline-invalid-ionic-md-ltr-light-Mobile-Chrome-linux.png:1]: Why is "Email" showing up twice here?

File: core/src/components/input/test/shape/input.e2e.ts-snapshots/input-shape-rectangular-fill-outline-custom-ionic-md-ltr-light-Mobile-Chrome-linux.png
Comment (brandyscarney) [inline core/src/components/input/test/shape/input.e2e.ts-snapshots/input-shape-rectangular-fill-outline-custom-ionic-md-ltr-light-Mobile-Chrome-linux.png:1]: Why is "Email" showing up twice here?

File: core/src/components/input/test/shape/input.e2e.ts-snapshots/input-shape-rectangular-fill-outline-ionic-md-ltr-light-Mobile-Chrome-linux.png
Comment (brandyscarney) [inline core/src/components/input/test/shape/input.e2e.ts-snapshots/input-shape-rectangular-fill-outline-ionic-md-ltr-light-Mobile-Chrome-linux.png:1]: Why is "Email" showing up twice here?

File: core/src/components/input/test/shape/input.e2e.ts-snapshots/input-shape-round-fill-outline-custom-ionic-md-ltr-light-Mobile-Chrome-linux.png
Comment (brandyscarney) [inline core/src/components/input/test/shape/input.e2e.ts-snapshots/input-shape-round-fill-outline-custom-ionic-md-ltr-light-Mobile-Chrome-linux.png:1]: Why is "Email" showing up twice here?

File: core/src/components/input/test/shape/input.e2e.ts-snapshots/input-shape-round-fill-outline-ionic-md-ltr-light-Mobile-Chrome-linux.png
Comment (brandyscarney) [inline core/src/components/input/test/shape/input.e2e.ts-snapshots/input-shape-round-fill-outline-ionic-md-ltr-light-Mobile-Chrome-linux.png:1]: Why is "Email" showing up twice here?

File: core/src/components/input/test/shape/input.e2e.ts-snapshots/input-shape-soft-fill-outline-ionic-md-ltr-light-Mobile-Chrome-linux.png
Comment (brandyscarney) [inline core/src/components/input/test/shape/input.e2e.ts-snapshots/input-shape-soft-fill-outline-ionic-md-ltr-light-Mobile-Chrome-linux.png:1]: Why is "Email" showing up twice here?

File: core/src/components/input/test/size/input.e2e.ts-snapshots/input-size-large-outline-ionic-md-ltr-light-Mobile-Chrome-linux.png
Comment (brandyscarney) [inline core/src/components/input/test/size/input.e2e.ts-snapshots/input-size-large-outline-ionic-md-ltr-light-Mobile-Chrome-linux.png:1]: Why is "Email" showing up twice here?

File: core/src/components/input/test/size/input.e2e.ts-snapshots/input-size-large-outline-round-ionic-md-ltr-light-Mobile-Chrome-linux.png
Comment (brandyscarney) [inline core/src/components/input/test/size/input.e2e.ts-snapshots/input-size-large-outline-round-ionic-md-ltr-light-Mobile-Chrome-linux.png:1]: Why is "Email" showing up twice here?

File: core/src/components/input/test/size/input.e2e.ts-snapshots/input-size-medium-outline-ionic-md-ltr-light-Mobile-Chrome-linux.png
Comment (brandyscarney) [inline core/src/components/input/test/size/input.e2e.ts-snapshots/input-size-medium-outline-ionic-md-ltr-light-Mobile-Chrome-linux.png:1]: Why is "Email" showing up twice here?

File: core/src/components/input/test/size/input.e2e.ts-snapshots/input-size-medium-outline-round-ionic-md-ltr-light-Mobile-Chrome-linux.png
Comment (brandyscarney) [inline core/src/components/input/test/size/input.e2e.ts-snapshots/input-size-medium-outline-round-ionic-md-ltr-light-Mobile-Chrome-linux.png:1]: Why is "Email" showing up twice here?

File: core/src/components/input/test/size/input.e2e.ts-snapshots/input-size-xlarge-outline-ionic-md-ltr-light-Mobile-Chrome-linux.png
Comment (brandyscarney) [inline core/src/components/input/test/size/input.e2e.ts-snapshots/input-size-xlarge-outline-ionic-md-ltr-light-Mobile-Chrome-linux.png:1]: Why is "Email" showing up twice here?

File: core/src/components/input/test/size/input.e2e.ts-snapshots/input-size-xlarge-outline-round-ionic-md-ltr-light-Mobile-Chrome-linux.png
Comment (brandyscarney) [inline core/src/components/input/test/size/input.e2e.ts-snapshots/input-size-xlarge-outline-round-ionic-md-ltr-light-Mobile-Chrome-linux.png:1]: Why is "Email" showing up twice here?

File: core/src/components/input/test/slot/input.e2e.ts-snapshots/input-slots-label-stacked-fill-outline-ionic-md-ltr-light-Mobile-Chrome-linux.png
Comment (brandyscarney) [inline core/src/components/input/test/slot/input.e2e.ts-snapshots/input-slots-label-stacked-fill-outline-ionic-md-ltr-light-Mobile-Chrome-linux.png:1]: Why is "Email" showing up twice here?

File: core/src/components/input/test/states/input.e2e.ts-snapshots/input-disabled-outline-ionic-md-ltr-light-Mobile-Chrome-linux.png
Comment (brandyscarney) [inline core/src/components/input/test/states/input.e2e.ts-snapshots/input-disabled-outline-ionic-md-ltr-light-Mobile-Chrome-linux.png:1]: Why is "Email" showing up twice here?

File: core/src/components/input/test/states/input.e2e.ts-snapshots/input-focused-ionic-md-ltr-light-Mobile-Chrome-linux.png
Comment (brandyscarney) [inline core/src/components/input/test/states/input.e2e.ts-snapshots/input-focused-ionic-md-ltr-light-Mobile-Chrome-linux.png:1]: Why is "Email" showing up twice here?

File: core/src/components/input/test/states/input.e2e.ts-snapshots/input-readonly-outline-ionic-md-ltr-light-Mobile-Chrome-linux.png
Comment (brandyscarney) [inline core/src/components/input/test/states/input.e2e.ts-snapshots/input-readonly-outline-ionic-md-ltr-light-Mobile-Chrome-linux.png:1]: Why is "Email" showing up twice here?

**Review by brandyscarney — CHANGES_REQUESTED** [review]
I left comments on all of the screenshots showing an invalid diff.

File: core/src/components/input/input.tsx [hunk 1]
```diff
@@ -725,7 +736,7 @@ export class Input implements ComponentInterface {
     const theme = getIonTheme(this);
     const hasOutlineFill = this.fill === 'outline';
 
-    if (hasOutlineFill && theme === 'md') {
+    if (hasOutlineFill && theme !== 'ios') {
```
[the remark this answers is not in the document]
Comment (BenOsodrac) [reply core/src/components/input/input.tsx]: Ahh sorry, stupid mistake. I changed that for testing and then forgot to remove it. Then when looking at snaphots, for some reason I thought the double label was some stacking label test :)

[the remark this answers is not in the document]
Comment (BenOsodrac) [reply core/src/components/input/input.tsx]: @brandyscarney I reverted it and updated snapshots!

**Review by joselrio — APPROVED** [review]

**Review by brandyscarney — APPROVED** [review]
Looks good! 👍

**Pull request merged**